answers-syscall.txt

```diff
@@ -0,0 +1,5 @@
+usertrap
+exec syscall
+user mode, since SPP bit is 0
+a3
```

Comment:
what's the answer to:
"Why does the kernel crash? Hint: look at figure 3-3 in the text; is address 0 mapped in the kernel address space? Is that confirmed by the value in scause above?"